src/components/DateInput/index.css

```diff
@@ -25,7 +25,7 @@
   border: none;
   border-radius: 0;
   box-shadow: inset 0 0 0 1px var(--black),
-    inset 0.25rem 0.25rem color-mod(var(--black) tint(80%));
+    inset 0 0.25rem color-mod(var(--black) tint(80%));
```

Comment:
Do we want to consider alpha for drop shadows? We can still use tints for faux borders.

Reply:
I'm not fussed, would be interested to know what benefit that would provide.

Reply:
Dropshadows now use alpha, with the exception of Submit which will be handled in a separate issue.